packages/libs/react-components/src/components/Grid/GridItem.tsx

```diff
@@ -0,0 +1,47 @@
+import { IThemeColors, styled } from './../../styles';
+
+import React, { FC, ReactNode } from 'react';
+
+const BaseItem = styled('div', {
+  gridColumnStart: 'auto',
+});
+
+export type IColSpan = number | 'auto';
```

Comment:
I think we should only prepend an I when it's an actual interface. If one wants to extend it because "it's an interface" it doesn't work. We could do `ColSpanType` if you're not happy with just `ColSpan` (but I think `ColSpan` is fine

Reply:
Yeah, I also think it's better to only prefix it if it is an Interface as it indicates that it is extendable. My preference is to go with `ColSpan`, but `ColSpanType` is also fine

Reply:
I like the ColsSpanType